recipes/renode-cli/recipe.yaml

```diff
@@ -0,0 +1,86 @@
+context:
+  name: renode-cli
+  version: "1.15.3"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/renode/renode/releases/download/v${{ version }}/renode_${{ version }}_source.tar.xz
+    sha256: 40c50afe8db86a9c63f4a86b93116a6babd43bf241cfaac47b2c0d2cbc2b15a8
+    patches:
+      - patches/update-IronPython-Renode_NET.csproj.patch
+      - patches/update-cores.template_NET.csproj.patch
+      - patches/update-IronPython-Unittest_NET.csproj.patch
+      # - patches/fix-from-master-Renode_NET.sln.patch # rattler seems to choke on leading tabs
+      - patches/fix-from-master-TermSharp_NET.csproj.patch
```

Comment:
I think it would be better to convert this patch to a simple `sed` replacement since it's a temporary fix for a dependency version that will probably be updated in a later release.  We can add a comment above the `sed` statement explaining that it can be removed once there is a new release that fixes the version.

Reply:
I moved them into a patch so that when a new version is released, it will make the patching fail and I won't forget to remove it, whereas, I would certainly forget the `sed`.

Reply:
That's fair, let's leave them as patches then!

Reply:
Well, that was my motivation, but then the same can be done inside the script with a test on the PKG_VERSION, so I implemented that. I am working on your suggestions, I broke linux build somewhere and working on that currently, I should have an update soon for unix to transfer to my windows VM

Reply:
Done